archimedes/src/app/component/image/image.component.html

```diff
@@ -0,0 +1,67 @@
+<div *ngFor="let image of images" class="col-sm-6 col-md-4 col-xl-3">
+    <div class="card mb-3">
+        <a href="{{image.download_url}}" target="_blank">
+            <img class="card-img-top" src="{{image.thumbnail_url}}" alt="{{image.location}}">
+        </a>
+        <div class="card-block">
+
+            <p class="card-text" (click)="(selectedImage === image.id) ? selectedImage = 0 : selectedImage = image.id">
+                View Image Details
+            </p>
+
+            <ng-container *ngIf="selectedImage === image.id">
+
+                <p class="card-text" *ngIf="image.tags && image.tags.length">
```

Comment:
What do you think about wrapping all of the image information (Tags, location, source, etc) in a `small` class? That seems to help the readability a bit. Not sure if you're updating this info to be in a modal, but it could possibly help there too